### PR TITLE
Fix campaign calendar bars positioning

### PR DESCRIPTION
**Pull request by pekc83**

### CODE REVIEW

**Comment by pekc83** [comment]
Closes #40

File: src/app/module-viewer/riverside-step-template/templates/campaign-calendar-template/campaign-calendar/campaign-calendar.component.html [hunk 1]
```diff
@@ -48,8 +48,11 @@ <h5 class="mt-3">{{year}}</h5>
       <table class="table table-bordered mt-2 position-relative">
         <thead>
         <tr>
-          <th scope="col" style="width: 28%">Campaign</th>
-          <th scope="col" *ngFor="let month of months" style="width: 6%">{{month}}</th>
+          <th scope="col" class="label">Campaign</th>
+          <th scope="col"
+              *ngFor="let month of months"
+              [style.width.%]="getMonthCellWidthPercent(month, year)"
```
Comment (integry) [inline src/app/module-viewer/riverside-step-template/templates/campaign-calendar-template/campaign-calendar/campaign-calendar.component.html]: Could this be pre-calculated in TS? I'm not a fan of putting function calls in templates as it comes with a significant performance penalty. The same applies to the function calls in the next few lines.

Comment (integry) [reply src/app/module-viewer/riverside-step-template/templates/campaign-calendar-template/campaign-calendar/campaign-calendar.component.html]: Also, I still see the issue:

image

**Comment by integry** [comment]
In some cases it still doesn't exactly match the size by a couple of pixels. But we can fix that later, there are more important things to look at right now.

**Pull request merged**